release-notes-1.3.0.md

```diff
@@ -0,0 +1,7 @@
+
+#### Version Number
+- This image is now based off the [OpenSUSE Base Image](https://github.com/CAFapi/opensuse-base-image) which has the postgreSQL client pre-installed. This provides the ability to create databases during startup of any consuming containers.
```

Comment:
Version number token shouldn't be being removed.

Reply:
Sorry my mistake I put the text in the wrong place.